src/frontend/shared-components/src/components/table/Table.stories.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+import Table from "./Table";
+import { getTableTestData } from "../../modules/tableTestData";
+
+export default {
+  title: "Table",
+  component: Table
+};
+
+const tableData = getTableTestData();
+
+export const WithHeader = () => <Table table={tableData} />;
+
+export const WithoutHeader = () => (
+  <Table table={{ ...tableData, heading: "" }} />
+);
+
+export const WithBlueStripeStyle = () => (
```

Comment:
any reason these two dont have mobile?

Reply:
I could add one for the Without header, but nothing of the shape changes for withBlueStripes so didn't think I needed a story for that.

Reply:
gotcha